src/components/MultiSelect/index.js

```diff
@@ -156,6 +156,8 @@ const MultiSelect = React.forwardRef((props, ref) => {
         }
     };
 
+    const handleContainerClick = event => event.stopPropagation();
```

Comment:
we need to avoid stopPropagation since can be a user that listens for clicks in containers up in the tree, then these clicks will not get in its listener, we should follow the same approach of the drawer

Reply:
Fixed